client/js/next-step.js

```diff
@@ -1,66 +1,27 @@
-const hide = (element) => {
+var hide = function(element) {
   if (element) {
     element.classList.add('hide');
   }
 }
 
-const unhide = (element) => {
+var unhide = function(element) {
   if (element && element.classList.contains('hidden')) {
     element.classList.remove('hidden');
     element.classList.add('unhide');
   }
 }
 
-/*
```

Comment:
Is there not a progress bar at all or does it's progress get computed in some more natural way now?

Reply:
there's no progress bar anymore